src/componenets/Header/Header.scss

```diff
@@ -0,0 +1,35 @@
+* {
+	margin: 0;
+}
+
+.header_main-container {
+	background: #0a1612;
+	height: 14vh;
+	color: #ffffff;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+	align-content: center;
+	padding: 0 10px;
+}
+
+.header_welcome-text {
+	display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  font-size: 12px;
+}
+
+.header_main-h1 {
+	margin: auto;
+	font-size: 45px;
```

Comment:
I've been using rem units for font size so far, I can show you the sizes I've used and we can make a style guide or something. looks great!